model/client4.go

```diff
@@ -1795,8 +1795,8 @@ func (c *Client4) PermanentDeleteTeam(teamId string) (bool, *Response) {
 }
 
 // GetTeamMembers returns team members based on the provided team id string.
-func (c *Client4) GetTeamMembers(teamId string, page int, perPage int, etag string) ([]*TeamMember, *Response) {
-	query := fmt.Sprintf("?page=%v&per_page=%v", page, perPage)
+func (c *Client4) GetTeamMembers(teamId string, page int, perPage int, sort string, exclude_deleted_users bool, etag string) ([]*TeamMember, *Response) {
```

Comment:
Unfortunately, this is a breaking change (in a Go SemVer sense), and someting we can only do in v6.0. Let's expose this as a `GetTeamMembersWithoutDeletedUsers` if this functionality is required.

Reply:
It's not required per se, it was just more of a sanity check that the api4 parses our parameters correctly and doesn't error. I added what you recommended and reverted the changes I made and ran my tests with the new function. I did mention in a comment that I wasn't able to modify the original function due to it being a breaking change.